packages/template-retail-react-app/app/utils/site-utils.js

```diff
@@ -28,7 +28,7 @@ export const resolveSiteFromUrl = (url) => {
 
     // get the site identifier from the url
     // step 1: look for the site based on the site identifier (id or alias) from the url
-    site = sites.find((site) => site.id === siteRef || site.alias === siteRef)
+    site = sites.find((site) => site.id === siteRef || (site.alias && site.alias === siteRef))
```

Comment:
nit, this one liner is becoming hard to understand. I'd encourage to expand this into a multi-line block function, using if statements.

Reply:
🤔 , I think it is still readable. Adding a comment might give it a bit more context. Using if is a bit lengthy imo. 
```
site = sites.find((site) => {
   if (site.id === siteRef) return true
   if (site.alias && site.alias === siteRef) return true
})

```